sys-apps/firejail/files/firejail-0.9.62.4-nvidia-no-nogroups.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/src/firejail/profile.c b/src/firejail/profile.c
```

Comment:
Where did this patch come from? Please add references.

Reply:
Ah, sure, see:
https://github.com/netblue30/firejail/issues/3644
https://github.com/netblue30/firejail/commit/623e6821687b848d6bb6f78a9deb682746df24ce

I will update the .patch file to include those in comments.